src/Tizen.Network.Connection/Tizen.Network.Connection/ConnectionManager.cs

```diff
@@ -405,7 +405,8 @@ public static CellularProfile CreateCellularProfile(ConnectionProfileType type,
         /// </summary>
         /// <since_tizen> 3 </since_tizen>
         /// <value>Cellular network state.</value>
-        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <feature>http://tizen.org/feature/network.telephony</feature>
```

Comment:
Is it no need to network.get privilege?

Reply:
Yes. CellularState uses vconf to get the state (key: VCONFKEY_NETWORK_CELLULAR_STATE)
On the other hand, others (wifi, bt, ethernet) use dbus method call.

Reply:
<vconfkey type="int32" name="memory/dnet/cellular" readPriv="http://tizen.org/privilege/internal/default/public" writePriv="http://tizen.org/privilege/internal/default/platform">